docs/Reference/Certificates/tls.md

```diff
@@ -0,0 +1,49 @@
+--8<-- "References/abbreviations.md"
+
+## What is TLS?
+Transport Layer Security (TLS) is a protocol used to establish encrypted communication sessions over an IP network between an application and server. As the most widely-used encryption protocol, it secures email, instant messaging, voice over IP (VoIP), HTTPS, and more. Encrypted TLS sessions promote data security and prevent tampering, eavesdropping, and other malicious activity.
+
+The protocol involves: 
+
+* **Public-key Cryptography:** public-private key pairs generated by cryptographic algorithms
+* **Digital Certificates:** an electronic document used to establish identity of the holder with the use of the public key
+	
+## What is a TLS digital certificate?
+
+Electronic documents, called certificates, are issued by and purchased from Certificate Authorities (CA), organizations that comply with TLS industry regulations, requirements, and standards.
+
+When a client running a web application, such as a browser, connects to a server, the server sends a TLS certificate that can be used to verify the authenticity of a server. The web application uses certificates present on the client to establish a secure connection. 
+
+There are two ways to use TLS in the toolkit:
+
+* Purchase a certificate from a CA and inject it via a proxy application 
+* Generate a self-signed certificate with mesh commander
```

Comment:
**MeshCommander**

Maybe link them to it too? https://www.meshcommander.com/meshcommander